### PR TITLE
Android: add documentation for alternative payment methods

### PR DESCRIPTION
**Pull request by jlejoux**

### CODE REVIEW

File: source/payments/mobile-android-sdk.md.erb [hunk 1]
```diff
@@ -69,6 +69,90 @@ client.tokenize(card, new TokenCallback() {
 });
 ```
 
+## Alternative payment methods
+ProcessOut also supports alternative payment methods with a redirection flow.
+First of all, make sure an alternative payment method has been activated on your account.
```
Comment (manuel-huez) [inline source/payments/mobile-android-sdk.md.erb]: This should probably not be displayed as in sandbox mode the merchant will always have an APM enabled to test

File: source/payments/mobile-android-sdk.md.erb [hunk 2]
```diff
@@ -69,6 +69,90 @@ client.tokenize(card, new TokenCallback() {
...
+    }
+
+    @Override
+    public void onError(Exception e) {
+        e.printStackTrace();
+    }
+});
+```
+Note that you'll need to generate an invoice in your backend and retrieve that invoice ID in order to fetch all the active configurations available for that invoice.
+You'll also need to provide this invoice with a specific return url of this form: `your_app://processout.return`
```
Comment (manuel-huez) [inline source/payments/mobile-android-sdk.md.erb]: We might want to explain how this return URL is set within the SDK

File: source/payments/mobile-android-sdk.md.erb [hunk 3]
```diff
@@ -69,6 +69,90 @@ client.tokenize(card, new TokenCallback() {
...
+    setContentView(R.layout.activity_main);
+
+    Intent intent = getIntent();
+    Uri data = intent.getData();
+    if (data != null)
+        Log.d("PROCESSOUT", "TOKEN=" + data.getQueryParameter("token"));
+}
+```
+
+You should then send this token to your backend in order to complete the charge.
```
Comment (manuel-huez) [inline source/payments/mobile-android-sdk.md.erb]:
```suggestion
You should then send this token to your backend in order to complete the transaction.
```

File: source/payments/mobile-android-sdk.md.erb (hunk 2 above)
Comment (jlejoux) [reply source/payments/mobile-android-sdk.md.erb]: Isn't it supposed to be done on their backend?

**Comment by jlejoux** [comment]
replaced by https://github.com/processout/documentation/pull/30

**Pull request closed without merging**